joomla-extensions/patchtester
# Remove language code prefix like "en-GB." from language files' names for Joomla 4 and other language file fixes

Pull Request for Issues #265 and #263 .

#### Summary of Changes

- Remove language code prefix, e.g. "en-GB." from name of language files like it was done for the CMS in the 4.0-dev branch.
- Copy the `COM_PATCHTESTER_XML_DESCRIPTION` string from `.sys.ini` to `.ini`.
- Alpha order the new sting `COM_PATCHTESTER_PATCH_BREAKS_SITE` to the right place in the en-GB file.
- Copy the same new sting to the en-US file.
- Translate the same new string to German and add it to the de-DE file.

#### Testing Instructions

Install following patch tester 4 rc-1 package patched with the modifications of this Pull Request (PR): [https://test5.richard-fath.de/com_patchtester_4.0.0.rc2_pr-268.zip](https://test5.richard-fath.de/com_patchtester_4.0.0.rc2_pr-268.zip) on a Joomla 4 nightly build or current 4.0-dev branch.

Check that patchtester works without any language string problems which were not there without this PR.
